lib/node_modules/@stdlib/lapack/base/dgttrf/README.md

```diff
@@ -0,0 +1,308 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2025 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# dgttrf
+
+> Compute an `LU` factorization of a real tri diagonal matrix `A` using elimination with partial pivoting and row interchanges.
+
```

Comment:
Another option is to write an introduction which includes TeX equations to illustrate what this package does.

> Given the following square matrix `A`,

(insert equation of matrix)

> the `LU` factorization is defined as

(insert equation of LU)

> where `L` is ... and `U` is ...

and then can we describe in terms of the overwritten arrays below?